index.html

```diff
@@ -2440,8 +2440,19 @@ <h5>Conclusion</h5>
 <ul data-ucr-role="conclusion-notes">
 <!-- bullet-point notes summarizing implementation details / blocking issues.
      Add tags to the start of a point with an empty `<a data-ucr-role="tag"></a>` where the text content or href matches a tag `id` value. -->
-<li><a data-ucr-role="tag" href=""></a>
-  <!-- short summary of how the tag applies -->
+<li>
+<a data-ucr-role="tag" href="#accessibility-potential"></a>
+Web maps provided by existing reference tools
+tend to disregard accessibility best practices,
+and assistive technology users have poor to middling experiences with web maps,
+so much so that it is common for developers to hide them completely from assistive technology.
+A standard implementation of a map element
+has the potential to dramatically improve accessibility for wide range of assistive technology users.
```

Comment:
> has the potential to dramatically improve accessibility for wide range of assistive technology users.

I think the potential accessibility improvements goes beyond assistive technologies, including general user preferences in the OS/browser settings, keyboard accessibility, and default UI and styles - e.g. currently only Bing Maps Embed, Bing Maps Control API and Google Maps API add a visible `outline` when the map viewer receives focus and is pannable using arrow keys).

So perhaps it'd be better to word this in a way that doesn't necessarily restrict the accessibility improvements to ATs, e.g.:
"_has the potential to dramatically improve accessibility for users with different forms of imparity._"?

Reply:
In my head I count even mundane things such as keyboards and browser settings as AT, but I think your wording sounds better 😂  Update incoming!

Reply:
@nchan0154 It's time for me to sleep, so I'll stop using this assistive technology of mine and go to bed 😄

Reply:
Thank you for the late night review!